server/routerlicious/packages/services-telemetry/src/sanitizationLumberFormatter.ts

```diff
@@ -23,18 +22,24 @@ export class SanitizationLumberFormatter implements ILumberFormatter {
 
 	public transform(lumber: Lumber<string>): void {
 		if (lumber.logLevel === LogLevel.Error && lumber.exception) {
```

Comment:
LogLevel.Warning will also allow Lumberjack to include exceptions... Do we also want to sanitize the exceptions in Warning level logs?

https://github.com/microsoft/FluidFramework/blob/96730d15928a314e8f376a0dc1991b2f1c01f537/server/routerlicious/packages/services-telemetry/src/lumberjack.ts#L113-L116

Reply:
We could check directly if the lumber object has an exception and sanitize it either way. I'm uncertain if sensitive data is getting leaked at warning level as well (to my understanding, leaks occur at error level). Let me know what you guys think!
cc @arikt-ms

Reply:
I also think that most of the leaks have to do with error LogLevel, but since it is possible that Lumberjack be emitted with warning LogLevel while still including an exception, I think it could be worth removing this part of the checl: `lumber.logLevel === LogLevel.Error &&` and just simply checking if the exception exists

Reply:
Done!